frontend-react/src/components/shared/Table/Table.module.scss

```diff
@@ -0,0 +1,53 @@
+@use 'uswds-core' as *;
+
+.reportstreamTable {
+  :global {
+    // Since these are USDS overrides, we use the :global property so we can
+    // locally scope everything within the CSS Module underneath .reportstreamTable
+    // without having to create new SCSS classes, and simply made overrides.
+    .usa-table thead th {
+        cursor: pointer;
+        user-select: none;
+        background-color: color('gray-5');
+    }
+
+    .usa-table thead th.active-col-header {
+        background-color: color('blue-30');
+    }
+
+    .usa-table td.active-col-td {
+        background-color: color('blue-10');
+    }
+
+    .usa-table thead .column-header-title-container {
+        display: flex;
+        align-items: center;
+    }
+
+    .usa-table thead .column-header-title-container p {
+        font-size: 14px;
+        margin: 0;
+    }
+
+    .usa-table thead .column-header-title-container svg {
+        flex: none;
+        margin-left: 2px;
+        height: 28px;
```

Comment:
Can we not use `size` for the Icon component?  I think we should align with the given sizes we have rather than introducing custom ones all the time.

Reply:
The size only goes down to `size={3}`, when Jillian's design is really `2.5` or so. See here: https://designsystem.digital.gov/components/icon/#component-variants-icon

We can definitely talk about changing the icons to be inline with USWDS standards! I'll make that change for now.